packages/gcf-utils/src/logging/trigger-info-builder.ts

```diff
@@ -63,47 +66,72 @@ export function buildTriggerInfo(
-    triggerInfo.trigger.github_delivery_guid = github_delivery_guid;
+    triggerInfo.trigger.github_delivery_guid = githubDeliveryGUID;
   }
 
   if (triggerType === TriggerType.GITHUB) {
-    const sourceRepo = requestBody['repository'] || {};
-    const repoName: string = sourceRepo['name'] || UNKNOWN;
+    const webhookProperties = {
+      trigger_source_repo: getRepositoryDetails(requestBody),
+      trigger_sender: requestBody.sender?.login || UNKNOWN,
+      payload_hash: getPayloadHash(requestBody),
+      github_event_type: getEventTypeDetails(
+        githubEventName,
+        requestBody.action
+      ),
+    };
+    triggerInfo.trigger = {...webhookProperties, ...triggerInfo.trigger};
+  }
 
-    const repoOwner = sourceRepo['owner'] || {};
-    const ownerName: string = repoOwner['login'] || UNKNOWN;
-    const ownerType: string = repoOwner['type'] || UNKNOWN;
+  return triggerInfo;
+}
 
-    const sender = requestBody['sender'] || {};
-    const senderLogin: string = sender['login'] || UNKNOWN;
+/**
+ * Parses GitHub event's source repository details
+ * @param requestBody the body of the incoming GitHub Webhook request
+ */
+function getRepositoryDetails(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  requestBody: {[key: string]: any}
+): {
+  repo_name: string;
+  owner: string;
+  owner_type: string;
+  url: string;
+} {
+  const UNKNOWN = 'UNKNOWN';
 
-    const repoIsKnown = repoName !== UNKNOWN && ownerName !== UNKNOWN;
-    const url: string = repoIsKnown
-      ? `https://github.com/${ownerName}/${repoName}`
-      : UNKNOWN;
+  const sourceRepo = requestBody['repository'] || {};
+  const repoName: string = sourceRepo['name'] || UNKNOWN;
 
-    const payload_hash = getPayloadHash(requestBody);
+  const repoOwner = sourceRepo['owner'] || {};
+  const ownerName: string = repoOwner['login'] || UNKNOWN;
+  const ownerType: string = repoOwner['type'] || UNKNOWN;
 
-    const webhookProperties = {
-      trigger_source_repo: {
-        repo_name: repoName,
-        owner: ownerName,
-        owner_type: ownerType,
-        url: url,
-      },
-      trigger_sender: senderLogin,
-      payload_hash: payload_hash,
-    };
+  const repoIsKnown = repoName !== UNKNOWN && ownerName !== UNKNOWN;
+  const url: string = repoIsKnown
+    ? `https://github.com/${ownerName}/${repoName}`
+    : UNKNOWN;
 
-    triggerInfo.trigger = {...webhookProperties, ...triggerInfo.trigger};
-  }
+  return {
+    repo_name: repoName,
+    owner: ownerName,
+    owner_type: ownerType,
+    url: url,
+  };
+}
 
-  return triggerInfo;
+/**
+ * Returns a description of the GitHub Event type
+ * @param requestBody the body of the incoming GitHub Webhook request
+ */
+function getEventTypeDetails(eventName: string, actionValue: string): string {
+  eventName = eventName === '' ? 'UNKNOWN' : eventName;
+  return `${eventName}${actionValue ? `: ${actionValue}` : ''}`;
```

Comment:
I might make this concatenate using a `.` to match what others do (like probot). `issue.labeled` or `pull_request.merged` etc.